### PR TITLE
Structure change under "all_bes_repos"

### PR DESCRIPTION
**Pull request by asa1997**
@panickervinod 
The structure of the repo details has been changed under "all_bes_repos"

"all_bes_repos": [                                                                  
                {
                    "id": 391939220,
                    "name": "prometheus",
                    "url": "https://github.com/Be-Secure/prometheus"
                }
                
 ]

### CODE REVIEW

File: bes_theme/assets/data/OSSP-master.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        "cvedetails": {                                                  
+            "bes_cve_details_id": "",                      
+            "cvedetails_product_id": "61503",                      
+            "cvedetails_vendor_id": "20905"
+        }, 
+        "project_repos": {
+            "main_github_url": "https://github.com/prometheus/prometheus",
+            "main_bes_url": "https://github.com/Be-Secure/prometheus",
+            "all_projects": {                                                                         
+                "prometheus": "https://github.com/prometheus/prometheus",
```
Comment (panickervinod) [inline bes_theme/assets/data/OSSP-master.js:16]: this should be same as "all_bes_repos"

File: bes_theme/assets/data/OSSP-master.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+        ],
+        "tags": {
+            "IND": [
+                "ALL"
+            ],
+            "SD": [
+                "SD-SOC"
+            ],
+            "COM": {
```
Comment (panickervinod) [inline bes_theme/assets/data/OSSP-master.js:120]: "Community led" , is a valid label

**Comment by jobyko** [comment]
Correction needed for label
eg. there is direct label available for CNCF projects, COM-F-CNCF

**Review by jobyko — CHANGES_REQUESTED** [review]
Correction needed for label
eg. there is direct label available for CNCF projects, COM-F-CNCF

**Pull request merged**